addons/bus/models/bus.py

```diff
@@ -154,6 +157,23 @@ def _bus_last_id(self):
         return last.id if last else 0
 
 
+class BusSendableMixin(models.AbstractModel):
+    _name = 'bus.sendable'
+    _description = 'Can send messages via bus.bus'
+
+    @api.model
+    def _bus_send(self, notification_type, message):
+        if not self:
+            return
+        self.env['bus.bus']._sendmany([
+            (record._bus_channel(), notification_type, message)
+            for record in self
```

Comment:
I don't think we want batch sending the same notification to multiple persons.

At least for us, whenever that happens, it is a sign that the notification should be sent on a more generic object instead (channel, group, ...).
Moreover we often have user-specific data in the message.

Do you have the use case?

Reply:
Use cases are in activities for instance, where we send `mail.activity/updated` notifications to all users concerned by the batch `create`